README.md

```diff
@@ -1,55 +1,85 @@
-# cloudflare-jwt-verifier
-Cloudflare worker that verifies incoming requests have a valid JSON Web Token (JWT) before the request is forwarded on to your infrastructure
+# Approov Quick Start - Cloudflare Worker
+
+Approov Cloudflare worker that verifies incoming requests have a valid [Approov](https://approov.io) JSON Web Token (JWT) before the request is forwarded on to your infrastructure or to a third party service.
+
+Approov is meant to be used when you want to ensure that only your mobile app can talk to your mobile API, and you can learn in more detail how this is done [here](https://approov.io/approov-in-detail.html).
 
-This use case is in conjunction with the [Approov](https://approov.io) service, although it could be modified to verify any JWT.
 
 ## Why?
 
-There are a couple of reasons for wanting to do this:
- 1. Prevent abuse of your unauthenticated APIs (scraping, malicious activity etc)
- 2. DDoS prevention
+There are a lot of [reasons](https://approov.io/approov-in-detail.html) for wanting to do this, and some of them are:
+
+* Allow the API to validate the request is from a genuine and untampered version of your mobile app
+* Prevent abuse of your APIs (scraping, malicious activity, etc)
+* No need for an API key in the app
+* [Approov Dynamic Certificate Pinning](https://approov.io/docs/latest/approov-usage-documentation/#approov-dynamic-pinning) with all the benefits from traditional implementations, but without their drawbacks
+* DDoS prevention
```

Comment:
But you may be using a CDN for this anyway - more specifically it prevents application DDoS attacks reaching the origin server which they might otherwise

Reply:
This DDoS prevention is a mention from Scoffable, and I left it because while Cloudflare excels at protecting from network level DDoS attacks we do it at the application level. 

So I can improve the message to be more clear where Cloudflare and Approov prevent DDoS.

Reply:
Done